geode-core/src/main/java/org/apache/geode/internal/cache/partitioned/PersistentBucketRecoverer.java

```diff
@@ -337,7 +343,7 @@ private void logWaitingForMembers() {
       Map<PersistentMemberID, Set<Integer>> allMembersToWaitFor = getMembersToWaitFor(false);
 
       boolean thereAreBucketsToBeRecovered =
-          (RedundancyLogger.this.allBucketsRecoveredFromDisk.getCount() > 0);
+          (PersistentBucketRecoverer.this.allBucketsRecoveredFromDisk.getCount() > 0);
```

Comment:
If you add a method on PersistentBucektRecoverer that returns "allBucektsRecoveredFromDisk.getCount()" then this line could just call that method without needing the prefix "PersistentBucketRecoverer.this."